TELCODOCS-2005: Telco Day 2 Troubleshooting and maintenance (Intro, General, Cluster) (#85939)
openshift/openshift-docs: modules/telco-troubleshooting-general-review-events.adoc

// Module included in the following assemblies:
//
// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-general-troubleshooting.adoc

:_mod-docs-content-type: PROCEDURE
[id="telco-troubleshooting-general-review-events_{context}"]
= Reviewing events

> Review events in a given namespace by using the CLI.

Reviewer: ```suggestion
You can review the events in a given namespace to find potential issues.
```

.Procedure

> . Check for events in your namespace:

Reviewer: ```suggestion
. Check for events in your namespace by running the following command:
```

+
[source,terminal]
----
$ oc get events -n <namespace> --sort-by=".metadata.creationTimestamp" <1>
----

> <1> Adding the `--sort-by=".metadata.creationTimestamp"` flag places the most recent events at the bottom of the output.

Reviewer: ```suggestion
<1> Adding the `--sort-by=".metadata.creationTimestamp"` flag places the most recent events at the end of the output.
```

. Optional: If the events within your specified namespace do not provide enough information, expand your query to all namespaces by running the following command:
+
[source,terminal]
----
$ oc get events -A --sort-by=".metadata.creationTimestamp" <1>
----

> <1> The `--sort-by=".metadata.creationTimestamp"` flag places the most recent events at the bottom of the output.

Reviewer: ```suggestion
<1> The `--sort-by=".metadata.creationTimestamp"` flag places the most recent events at the end of the output.
```

+
To filter the results of all events from a cluster, you can use the `grep` command.
For example, if you are looking for errors, the errors can appear in two different sections of the output: the `TYPE` or `MESSAGE` sections.

> With `grep`, you can search for keywords, such as `error` or `failed`.

Reviewer: ```suggestion
With the `grep` command, you can search for keywords, such as `error` or `failed`.
```
